python/importer/remove_old_data.py

```diff
@@ -0,0 +1,22 @@
+import psycopg2
+import logging
+from common.utils import get_conn_params
+
+def main():
+    logger = logging.getLogger('importer')
+
+    conn = psycopg2.connect(**get_conn_params())
+    try:
+        with conn:
+            with conn.cursor() as cur:
+                logger.info("Removing data.")
+                # Remove data that is older than 2 weeks
+                cur.execute("DELETE FROM hfp.observed_journey WHERE oday < now() - interval '2 week'")
```

Comment:
Idea: after each `DELETE`, log the number of rows deleted, so we can spot from the logs if unusually high or low number of rows have been deleted. This could be easily done with the cursor's [`rowcount`](https://www.psycopg.org/docs/cursor.html#cursor.rowcount) attribute that tells how many rows where affected by the last `execute` command. Like so:

```python
cur.execute("DELETE FROM hfp.observed_journey WHERE oday < now() - interval '2 week'")
logger.info(f"{cur.rowcount} rows deleted.")
```

Unfortunately, this idea wouldn't tell the number of rows deleted by the `CASCADE` in `hfp.hfp_point`... well, at least the logging message could be something like `logger.info(f"{cur.rowcount} rows deleted from hfp.observed_journey, and all related rows in hfp.hfp_point.")`.